app/models/setting.rb

```diff
@@ -233,7 +233,10 @@ def validate_host_owner
     owner_type_and_id = value
     return if owner_type_and_id.blank?
     owner = OwnerClassifier.new(owner_type_and_id).user_or_usergroup
-    errors.add(:value, _("Host owner is invalid")) if owner.nil?
+    if owner.nil?
+      errors.add(:value, _("Host owner is invalid. " \
+                           "Please ensure the input follows the format 'ID-Users/Usergroups'."))
```

Comment:
If you're doing this, I think it should be clear that `ID-` is also the prefix for `Usergroups`. Right now that isn't obvious to me. And that `ID` must be replaced with an actual value but `Users` or `Usergroups`. I'd either go with `${id}-[Users|Usergroups]` or `$id-$type where type is Users or Usergroups`.